dev/build/windows/makecoq_mingw.sh

```diff
@@ -267,7 +267,7 @@ function get_expand_source_tar {
       return 1
     fi
   else
-    logn untar tar xvaf "$TARBALLS/$name.$3" --strip $strip
+    logn untar tar xaf "$TARBALLS/$name.$3" --strip $strip
   fi
```

Comment:
I put in the v for a reason. I like to see what files are expanded. Sometimes strange things happen in CI even when untaring, and then it is good to know how far the untar get. I generally don't think that long log files are a problem, as long as they are not so long that downloading the raw log file takes considerable time.

Reply:
They are definitively a problem as they are impossible to make a sense of, cf #8045